docs/WRAPPER_REACTQUERY.md

```diff
@@ -0,0 +1,25 @@
+# react-query wrapper
+
+If you want to use MMKV with [react-query](https://tanstack.com/query/v4/docs/plugins/createSyncStoragePersister), create the following `clientPersister` object:
```

Comment:
Maybe also mention that you need to install the `@tanstack/query-sync-storage-persister` package?